### ISSUE TITLE
Re#736 - Marks for new and updated

### ISSUE THREAD

**Issue by wesruv** [backdrop/backdrop-issues#760]
Make new & updated markers look like this:

![mark](https://cloud.githubusercontent.com/assets/397895/7218040/409e7e76-e60a-11e4-9524-ebbd1f48a2bf.png)

See master issue for instructions/guidelines for implementation:
https://github.com/backdrop/backdrop-issues/issues/736

**Comment by Graham-72** [comment]
In PR 819 I have modified the styles for class .marker but the commit has failed Travis CI

**Comment by wesruv** [comment]
Assigning to @Graham-72 to prevent confusion

**Comment by Graham-72** [comment]
I have created a new PR at backdrop/backdrop/pull/828

**Comment by jenlampton** [comment]
Thanks @Graham-72 !

Sorry to do this to you, but Darius and I had a quick review of all the assumptions I made to unblock issues, and apparently I got this one wrong :(  He would prefer to use the colors from the green status message, light green bg and the dark gray text. Do you want to give that a whirl?

**Comment by Graham-72** [comment]
@jenlampton - No problem (I hope!). I will do it today.

**Comment by Graham-72** [comment]
@jenlampton @quicksketch Thanks for comments on my PR. I have revised it and I hope I have done this the right way. backdrop/backdrop#828

**Comment by quicksketch** [comment]
Looks pretty good to me! Some times it's helpful to squash the commits together into a single one. You can do this by modifying the last commit with `git commit --amend`, or afterwards with `git rebase -i` (see https://backdropcms.org/develop/pull-requests#clean-requests), but in this case, both commit messages are good and they modify the same lines, so it's not necessary to squash them together. Thanks @Graham-72! I'll double-check the appearance of these locally before merging.

**Comment by quicksketch** [comment]
Merged https://github.com/backdrop/backdrop/pull/828. Thanks!
